wcc-contentful-app/spec/tasks/validate_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'rake'
+require 'rails_helper'
+require 'csv'
+
+RSpec.describe 'tasks/validate' do
+  Rake::Task.define_task(:environment)
+  Rake.application.rake_require 'tasks/validate'
+
+  def act
+    Rake::Task['wcc_contentful:validate'].invoke
+    Rake::Task['wcc_contentful:validate'].reenable
+  end
+
+  # TODO: https://github.com/watermarkchurch/wcc-contentful/issues/40
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:book:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/watermarkchurch/wcc-contentful/blob/b297f02f42968b0f28dc0863fe9e74e354e2de9a/wcc-contentful-app/spec/tasks/validate_spec.rb#L16"></span>TODO added in <a href="https://github.com/watermarkchurch/wcc-contentful/blob/b297f02f42968b0f28dc0863fe9e74e354e2de9a/wcc-contentful-app/spec/tasks/validate_spec.rb">wcc-contentful-app/spec/tasks/validate_spec.rb</a> referencing <a href="https://github.com/watermarkchurch/wcc-contentful/issues/40">https://github.com/watermarkchurch/wcc-contentful/issues/40</a></td>
    </tr>
  </tbody>
</table>